deploy/crds/opentelemetry.io_opentelemetrycollectors_crd.yaml

```diff
@@ -68,6 +68,1224 @@ spec:
               description: ServiceAccount indicates the name of an existing service
                 account to use with this instance.
               type: string
+            volumeMounts:
```

Comment:
Would it be a good idea to remove descriptions from the CRD, as with jaeger operator, to avoid potential size issues in the future?

Reply:
The problem with the Jaeger Operator is that we support Volumes/VolumeMounts at several levels, making our CRD repeat them quite a few times. For this one here, we have it only once, so, we can keep it for now. If we do end up repeating those fields I'll remove the description, but hopefully there will be a proper solution by then :-)